src/nvim/path.c

```diff
@@ -1553,21 +1552,19 @@ int path_with_url(char_u *fname)
  */
 int vim_isAbsName(char_u *name)
```

Comment:
this can be converted to a bool; all call sites follow the scheme `!?vim_isAbsName(param`.